book-store/src/redux/books/books.js

```diff
@@ -0,0 +1,31 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = [];
+
+const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
```

Comment:
* Kindly implement the add book functionality as currently, i can't add a book please fix it as it is required for this milestone [Check project requirement as a refresher ](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_create_and_remove_book.md#:~:text=In%20your%20React%20component%20responsible%20for%20adding%20new%20books%20set%20the%20data%20inputs%20in%20the%20local%20React%20state%20(set%20title%20and%20author%20-%20NOTE%3A%20categories%20and%20comments%20are%20NOT%20part%20of%20this%20step).%20Once%20your%20new%20book%20object%20is%20ready%20to%20be%20submitted%20to%20Redux%20store%2C%20dispatch%20a%20corresponding%20action%2C%20i.e.%3A)

Reply:
Fixed